lib/support/tcpdump.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { spawn } = require('child_process');
+const get = require('lodash.get');
+const mkdirp = require('mkdirp');
```

Comment:
You can pass on the storage manager from Iteration and then create the dir with https://github.com/sitespeedio/browsertime/blob/master/lib/support/storageManager.js#L54 - then we keep making dirs in the StorageManager